packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -4001,142 +4001,97 @@ describe('StripeHelper', () => {
   });
 
   describe('expandResource', () => {
-    it('returns the same resource if it is not an id string', async () => {
-      const actual = await stripeHelper.expandResource(
-        customer1,
+    let customer;
+
+    beforeEach(() => {
+      customer = deepCopy(customer1);
+    });
+
+    it('expands the customer', async () => {
+      stripeFirestore.retrieveAndFetchCustomer = sandbox
+        .stub()
+        .resolves(deepCopy(customer));
+      stripeFirestore.retrieveCustomerSubscriptions = sandbox
+        .stub()
+        .resolves(deepCopy(customer.subscriptions.data));
+      const result = await stripeHelper.expandResource(
+        customer.id,
         CUSTOMER_RESOURCE
       );
-      assert.deepEqual(actual, customer1);
+      // Note that top level will mismatch because subscriptions is copied
+      // without the object type.
```

Comment:
What do you mean here by object type? Can you clarify why a deepEqual for result and customer won't work?

Reply:
These are existing tests that I haven't changed in this commit. They're just being moved. Object type is a field in the Stripe object being compared.